webservices/partition/base.py

```diff
@@ -266,46 +267,20 @@ def rename_child(cls, cycle):
             db.engine.execute(cmd)
 
     @classmethod
-    def refresh_children(cls):
-        """Loops through all child tables and refreshes them.
-
-        Returns a list of tuples that contain success values and messages with
-        details.
-        """
-
+    def process_queues(cls):
         queue_old = utils.load_table(cls.queue_old)
         queue_new = utils.load_table(cls.queue_new)
-        cycles = get_cycles()
-        output_messages = []
-
-        for cycle in cycles:
-            output_messages.append(
-                cls.refresh_child(cycle, queue_old, queue_new)
-            )
-
-        return output_messages
 
-    @classmethod
-    def refresh_child(cls, cycle, queue_old, queue_new):
-        """Refreshes a single child table and returns a tuple with a success
-        value (0 for success, 1 for failure) and message with details.
-        """
-
-        start, stop = cycle - 1, cycle
         output_message = (0, '')
-        name = '{base}_{start}_{stop}'.format(
-            base=cls.base_name, start=start, stop=stop
-        )
-        child = utils.load_table(name)
+        name = '{base}_master'.format(base=cls.base_name)
+        master_table = utils.load_table(name)
         connection = db.engine.connect()
         transaction = connection.begin()
 
         try:
-            delete_select = sa.select([queue_old.c.get(cls.primary)]).where(
-                queue_old.c.two_year_transaction_period.in_([start, stop])
-            )
-            delete = sa.delete(child).where(
-                child.c.get(cls.primary).in_(delete_select)
+            delete_select = sa.select([queue_old.c.get(cls.primary)])
+            delete = sa.delete(master_table).where(
+                master_table.c.get(cls.primary).in_(delete_select)
```

Comment:
Interesting!  So you're removing the specific processing for each child table and shifting the updates from the queues to the master table itself?